.github/workflows/main/options.posix

```diff
@@ -42,6 +42,7 @@ BOOST_INCLUDE_PATH = includes
 BOOST_LIB_SUFFIX = ""
 
 OPENEXR_INCLUDE_PATH = includes
+ILMBASE_INCLUDE_PATH = includes + "\\imath"
```

Comment:
Why are we using a Windows style path in the Posix config?

Reply:
The CopyPaste giveth, and the CopyPaste taketh away.

Fixed and squashed into ffe6f9e62e1fffc93e112b723ecab7d241282281

Reply:
Good idea, that is a bit cleaner and clearer. Fixed and squashed into 246961d992ed23599997e7536389ababfe1c868e